packages/protocol/contracts/L1/tiers/TierProviderBase.sol

```diff
@@ -37,6 +37,17 @@ abstract contract TierProviderBase is ITierProvider {
             });
         }
 
+        if (_tierId == LibTiers.TIER_SGX2) {
```

Comment:
This the SGX verifier with "ontake" fork support?

Reply:
Yes